adr/0001-community-discussions.adoc

```diff
@@ -0,0 +1,70 @@
+= Setup Quarkus Github Discussions to enable the broader Community
+
+* Status: proposed
+* Date: 2021-06-26
+
+== Context and Problem Statement
+
+Quarkus community is growing and until now we've catered very well for core contributors and initial early consumers. 
+
+We have multiple communication channels: https://github.com/quarkusio/quarkus[issues], https://groups.google.com/g/quarkus-dev?pli=1[quarkus-dev mailing list], https://quarkusio.zulipchat.com[zulip], https://stackoverflow.com/questions/tagged/quarkus[stackoverflow]
+
+Isues are great for bugs/feature work. Mailing list for design conversations between developers, chat for watercooler style discussions and stackoverflow for user questions.
+
+This setup has issues though, some are:
+
+- zulip chat is used for a lot of users questions but none of that is easily searchable/discoverable so it is very synchronous,
+- people reported that they don't feel okey posting on quarkus-dev or zulips as it is seems focused on dev work and not so much about community events, jobs, conferences, etc.
+- its hard to monitor as contributor who wants to help answer/ask questions.
+
+How can we improve this situaton and enable the broader community to more easily ask questions and find answers - without it all be relying on just a few Quarkus core contributors?
+
+== Scenarios (optional)
+
+User wants to locate a answer to a question - zulip chats does not show up in google search; stackoverflow might but might not have an answer.
+
+Contributor want to arrange or attend an event around Quarkus - where do he post/look for info on that ?
+
```

Comment:
That's a good point - and it's more like a forum kind of thing. Do you have an example of GH Discussions that is used that way? In Mutiny, we use it as a SO clone.

Reply:
yes - I see github discussions big win for us to be more a forum with optional enablement of Q/A sections.

The "biggest" one is https://github.com/vercel/next.js/discussions with its show and tell 

even the default categories made by github has non q/a sections.